public/hooks/use_fetch_agentframework_traces.test.ts

```diff
@@ -39,13 +39,13 @@ describe('useFetchAgentFrameworkTraces hook', () => {
 
     services.http.get.mockResolvedValueOnce(traces);
     const { result, waitForNextUpdate } = renderHook(
-      () => useFetchAgentFrameworkTraces(traceId),
+      () => useFetchAgentFrameworkTraces(interactionId),
       wrapper
     );
 
     await waitForNextUpdate();
     expect(services.http.get).toHaveBeenCalledWith(
-      `/api/assistant/trace/${traceId}`,
```

Comment:
Nit: Should we update all `trace` wording to `interaction`? Just find still exists some API, hook, component are using `trace`.